src/Tizen.NUI/src/public/WindowSystem/TizenShell.cs

```diff
@@ -0,0 +1,132 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI.WindowSystem.Shell
+{
+    /// <summary>
+    /// Tizen Window System Shell.
+    /// This is a basic handle class for others in WsShell.
+    /// Others class in WsShell using this basic class to use TIzen Window System Shell.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class TizenShell : IDisposable
+    {
+        private IntPtr _tzsh;
+        private bool disposed = false;
+
+        /// <summary>
+        /// Throw error codes generated by tizen ws shell
+        /// </summary>
+        /// <param name="error"></param>
+        /// <since_tizen> 8 </since_tizen>
+        public void ErrorCodeThrow(int error)
```

Comment:
ErrorCodeThrow is always used with error code from internal function.
Wouldn't it be okay if it was internal rather than public?

Reply:
I'll fix it